static/js/tabpanel.js

```diff
@@ -0,0 +1,73 @@
+let timer;
+
+window.addEventListener('DOMContentLoaded', () => {
+  const tabs = document.querySelectorAll('[role="tab"]');
```

Comment:
Should this not be namespaced or it will apply to any other elements with a role of "tab" site-wide?

Reply:
Agree, good point

Reply:
This issue is still unactioned?

Reply:
Done